scss/mixins/_forms.scss

```diff
@@ -156,7 +156,7 @@
     // on focus set borders and labels to the validation color
 
     // Use the BS provided mixin for the bulk of the color
-    @include form-validation-state("valid", $label-color);
+    @include form-validation-state("valid", $label-color, "none");
```

Comment:
May you help me understand this change please?

Reply:
This update is required because bootstrap 4 expects extra param